app/components/layouts/components/suggestionList.tsx

```diff
@@ -79,29 +81,43 @@ const SuggestionList: React.SFC<SuggestionListProps> = props => {
         e.preventDefault();
         props.handleClickSuggestionKeyword(props.suggestionList[index]);
       }}
-      className={styles.keywordCompletionItem}
+      className={classNames({
+        [`${styles.keywordCompletionItem}`]: index != 0,
+        [`${styles.highLightKeywordCompletionItem}`]: index === 0,
+      })}
       onKeyDown={e => {
         handleArrowKeyInput(e, props.handleClickSuggestionKeyword, props.suggestionList[index]);
       }}
       key={`keyword_completion_${suggestion}${index}`}
       tabIndex={-1}
     >
-      <span dangerouslySetInnerHTML={{ __html: suggestion }} />
+      <span className={styles.keywordCompletionItemContext} dangerouslySetInnerHTML={{ __html: suggestion }} />
     </a>
   ));
-
+  highlightedList.length;
```

Comment:
What is the purpose of this line?

Reply:
That's my testing trace. My mistake! I will remove.